components/service/glean/src/test/java/mozilla/components/service/glean/storages/TimingDistributionsStorageEngineTest.kt

```diff
@@ -128,6 +135,39 @@ class TimingDistributionsStorageEngineTest {
             snapshot["telemetry.test_timing_distribution"]?.toJsonObject().toString())
```

Comment:
Let's also change the case in [this test](https://github.com/mozilla-mobile/android-components/blob/b49cd72c5c27ad335fd1885a358071abc2c70214/components/service/glean/src/test/java/mozilla/components/service/glean/storages/TimingDistributionsStorageEngineTest.kt#L89-L105). It is supposed to fail, but let's make sure it fails for the right reasons :)